packages/block-editor/src/components/block-alignment-toolbar/index.js

```diff
@@ -88,11 +83,6 @@ export function BlockAlignmentToolbar( {
 }
 
 export default compose(
-	withBlockEditContext( ( { clientId } ) => {
```

Comment:
Another interesting observation with migrating from higher-order components to hooks: With higher-order components, it's fairly easy to overlook when something is totally unused!

Reply:
Yes, I've often seen left over selectors :)